src/core/main.c

```diff
@@ -2487,18 +2493,28 @@ static int initialize_runtime(
                                 log_warning_errno(r, "Failed to copy os-release for propagation, ignoring: %m");
                 }
 
-                /* Clear ambient capabilities, so services do not inherit them implicitly. Dropping them does
-                 * not affect the permitted and effective sets which are important for the manager itself to
-                 * operate. */
-                (void) capability_ambient_set_apply(0, /* also_inherit= */ false);
-
                 break;
         }
 
         default:
                 assert_not_reached();
         }
 
+        /* The two operations on the ambient set are meant for a user serssion manager. They do not affect
+         * system manager operation, because by default it starts with an empty ambient set.
+         *
+         * Preserve the ambient set for later use with sd-executor processes. */
+        r = capability_get_ambient(original_ambient_set);
```

Comment:
given the apply call below ignores failures, this should too - logging loudly is fine